addons/event_sale/tests/test_event_sale.py

```diff
@@ -279,3 +280,22 @@ def test_cancel_so(self):
         self.assertEqual(event.seats_expected, 1)
         self.sale_order.action_cancel()
         self.assertEqual(event.seats_expected, 0)
+
+    def test_check_event_id_and_event_ticket_id(self):
+        so_1 = self.env['sale.order'].create({
```

Comment:
```suggestion
        so = self.env['sale.order'].create({
```

very nitpicky but if there's only one of each I think it's clearer if we don't number them. This way nobody will look for so_2 ^^